packages/tools/generator-component/generators/app/templates/test-utils/component-objects/f-skeleton.component.js

```diff
@@ -1,25 +1,15 @@
 const Page = require('@justeat/f-wdio-utils/src/page.object');
-const { COMPONENT } = require('./f-<%= name.class %>-selectors');
+const { COMPONENT } = require('./f-<%= name.default %>-selectors');
 
 module.exports = class <%= name.filename %> extends Page {
     constructor () {
         super('<%= storybook.componentCategory.toLowerCase().slice(0, -1) %>', '<%= name.default %>-component');
     }
 
-    // eslint-disable-next-line class-methods-use-this
     get component () {
-        // eslint-disable-next-line no-undef
         return $(COMPONENT);
     }
-
-    load () {
-        super.load(this.component);
-    }
-
-    open (url) {
-        super.open(url);
-    }
-
+    
     waitForComponent () {
         super.waitForComponent(this.component);
     }
```

Comment:
If you are bumping wdio to v0.5.0 then you might want this little extra sugar ...
```suggestion
    waitForComponent (timeoutMs) {
        super.waitForComponent(this.component, timeoutMs);
    }
```